### ISSUE TITLE
readme discord invalid

### ISSUE THREAD

**Issue by snqwq** [libretro/RetroArch#15568]
hey I was trying to join the discord for retro arch but then I saw the link was invalid just thought I would let you guys know

(keep up the good work btw 👍)

**Comment by hizzlekizzle** [comment]
Does this one work? If so, can you link to the nonfunctional one you tried? https://discord.com/invite/VZ2b7wghxR

**Comment by snqwq** [comment]
yes that one worked (thanks!) and the one that didnt function was https://discord.com/invite/27Xxm2h

btw it was at the bottom of the readme ([here](https://github.com/libretro/RetroArch#user-content-socials))

**Comment by hizzlekizzle** [comment]
ok, we'll change it there. Thanks for the heads-up!
